docs/src/wrapper.tsx

```diff
@@ -16,7 +16,17 @@ export const WrapPageElement: GatsbyBrowser['wrapPageElement'] = ({
 export const WrapRootElement: GatsbyBrowser['wrapRootElement'] = ({
   element,
 }) => (
-  <ThemeProvider theme={b2bTheme}>
+  <ThemeProvider theme={theme}>
     <MDXProvider components={mdxComponents}>{element}</MDXProvider>;
   </ThemeProvider>
 );
+
+/**
+ * Enforce reloading to update styles.
```

Comment:
Can you tell me why we need this?

Reply:
Webpack HMR does not recognize changes in the styles (e.g. when you update a theme and this causes changes in the CSS styles).

This makes sure the page automatically refreshes, so we don't have to do it manually.

Reply:
Thanks :)